### ISSUE TITLE
Prepare for 2022 release

### ISSUE THREAD

**Issue by onobc** [spring-cloud/stream-applications#311]
- [x] Branch main into 2021.1.x
- [x] Use main for SB3/SF6 work
- [x] Stop building 2020.0.x
- [x] Start building 2021.1.x
- [x] Merge parked PRs once we branch

Version number `2022.0.0` w/ rough GA date of early-mid December.

Once the above is complete we can start migrating `main` to use SB3/SF6.
